code/modules/spells/changeling/regenerative_stasis.dm

```diff
@@ -5,6 +5,8 @@
 	hud_state = "regenstasis"
 
 	spell_flags = NEEDSHUMAN | STATALLOWED
+	charge_max = 8 MINUTES
+	cooldown_min = 8 MINUTES
```

Comment:
this should be in the changelog

Reply:
it is, but I removed the other constraint on Regenerative Stasis. Please updoot.